initial_pose: 
  x: 3
  y: 4.5
  z: 1
  R_deg: 0
  P_deg: 0
  Y_deg: 0
  camera_deg: -82
collision_distance: 1.02